### PR TITLE
creating a zip with deployment artifacts and adding developer guide

### PR DESCRIPTION
**Pull request by Shehanir**
adding developer guide to build the image after changes added. 

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: distribution/pom.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.apim</groupId>
+    <artifactId>k8s-apim-operator-distribution</artifactId>
+    <version>0.6</version>
+    <name>WSO2 APIM k8s CRDs zip Components</name>
```
Comment (pubudu538) [inline distribution/pom.xml]: Let's change this to K8s APIM Operator Distribution.

File: distribution/pom.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                                <descriptor>assembly.xml</descriptor>
+                            </descriptors>
+                            <finalName>wso2am-k8s-crds-0.6</finalName>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
```
Comment (nalakas) [inline distribution/pom.xml]: remove unnecessary line

File: distribution/pom.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+</project>
```
Comment (nalakas) [inline distribution/pom.xml]: add a new line at the end of file

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    <name>WSO2 APIM Kubernetes APIM Operator</name>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>distribution</module>
+    </modules>
+</project>
+
+
+
```
Comment (nalakas) [inline pom.xml]: remove unnecessary line

File: apim-operator/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+#### Developer Guide
+
+**Building the image**
+
+The apim-operator was built using the operator-sdk CLI tool. you can follow the [user guide][operator_sdk_user_guide] of operator-sdk CLI tool for
```
Comment (malinthaprasan) [inline apim-operator/README.md]: Let's use the first letter capital when starting the sentence, eg: You can follow ..

File: distribution/assembly.xml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+                <exclude>scenarios/**</exclude>
+            </excludes>
+        </fileSet>
+        <fileSet>
+            <outputDirectory>/scenarios</outputDirectory>
+            <directory>../apim-operator/deploy/scenarios</directory>
+        </fileSet>
+    </fileSets>
+</assembly>
+
```
Comment (malinthaprasan) [inline distribution/assembly.xml]: Keep only a single empty line at the end of files.

File: distribution/pom.xml (hunk 2 above)
Comment (Shehanir) [reply distribution/pom.xml:52]: Fixed

File: distribution/pom.xml (hunk 3 above)
Comment (Shehanir) [reply distribution/pom.xml:54]: Fixed

File: pom.xml (hunk 4 above)
Comment (Shehanir) [reply pom.xml:35]: Fixed

File: apim-operator/README.md (hunk 5 above)
Comment (Shehanir) [reply apim-operator/README.md:5]: Resolved

File: pom.xml (hunk 4 above)
Comment (Shehanir) [reply pom.xml:35]: Fixed

File: distribution/assembly.xml (hunk 6 above)
Comment (Shehanir) [reply distribution/assembly.xml:52]: Fixed

**Pull request merged**